lib/crypto/ed25519/ed25519.go

```diff
@@ -44,6 +44,20 @@ type PublicKey ed25519.PublicKey
 // PublicKeyBytes is an encoded ed25519 public key
 type PublicKeyBytes [PublicKeyLength]byte
 
+// SignVerify verify signature given pubkey and msg
+func SignVerify(pubkey, sig, msg []byte) (bool, error) {
```

Comment:
Can you change this to use `publicKey, signature, message []byte` for readability? Thanks!

Reply:
```suggestion
func VerifySignature(pubkey, sig, msg []byte) (bool, error) {
```